master/docs/tutorial/docker.rst

```diff
@@ -69,10 +69,10 @@ You should now be able to go to http://localhost:8010 and see a web page similar
 .. image:: _images/index.png
    :alt: index page
 
-Click on the `Waterfall Display link <http://localhost:8010/#/waterfall>`_ and you get this:
+Click on "Builds" at the left to open the submenu and then `Builders <http://localhost:8010/#/builders>`_ to see that the worker you just started has connected to the master:
```

Comment:
sub-menu

Reply:
I see that sort of incorrect suggestion a lot from spell checkers, for some reason. But according to the [OED](https://en.oxforddictionaries.com/definition/submenu) and [Merriam-Webster](https://www.merriam-webster.com/dictionary/submenu) it's fine (with only MW listing "sub-menu" as a variant).

I prefer it without the hyphen, parallel to some other common words taking a "sub" prefix ("substation" over "sub-station", "substandard" over "sub-standard") but I'm not going to bikeshed, so take this fix if you feel strongly about it.

```suggestion
Click on "Builds" at the left to open the sub-menu and then `Builders <http://localhost:8010/#/builders>`_ to see that the worker you just started has connected to the master:
```